src/NzbDrone.Core/Download/Pending/PendingReleaseService.cs

```diff
@@ -368,6 +368,7 @@ private Queue.Queue GetQueueItem(PendingRelease pendingRelease, Lazy<DateTime> n
                 RemoteEpisode = pendingRelease.RemoteEpisode,
                 Timeleft = timeleft,
                 EstimatedCompletionTime = ect,
+                AddedOn = pendingRelease.Added, // Check that this is right
```

Comment:
I'm 60% sure this isn't right -- is a pending release one done in the calendar, which shouldn't have an AddedOn time yet?

Reply:
Knowing when it's added to pending would also be useful, so I think this is right, just needs the comment removed.